### PR TITLE
Avoid a crash if the supposed search alias exists as an index

### PR DESCRIPTION
**Pull request by leonardr**
I'm not sure if this is the right fix for https://github.com/NYPL-Simplified/server_core/issues/579, but it works and I figure it's a good place to start.

### CODE REVIEW

File: external_search.py [hunk 1]
```diff
@@ -80,13 +80,30 @@ def set_works_index_and_alias(self, current_alias):
...
+            found_index = False
+
+        if found_index:
+            # The alias exists as an index. This isn't supposed to happen,
+            # but we can avoid a meltdown now by doing nothing.
+            self.log.error(
+                'Supposed alias "%s" already exists, as an index.',
+                current_alias
+            )
+            return
```
Comment (aslagle) [inline external_search.py]: So after this we'd end up with self.works_index = None?

Comment (leonardr) [inline external_search.py]: That's a problem. It works when I run it for real (not sure how) but it makes tests fail.

Comment (aslagle) [inline external_search.py]: It may be able to read from the index without that but not write to the index.

**Comment by leonardr** [comment]
We've decided to fix the BPL index manually and address this problem in 2.0.

**Pull request closed without merging**